src/routes/components/HeroBanner.tsx

```diff
@@ -149,24 +156,68 @@ export default function HeroBannerPage() {
       />
 
       <ComponentContent tocCssQuery="goa-tab[open=true] :is(h2[id], h3[id])">
-
         <GoATabs>
           <GoATab heading="Code examples">
             {/*Hero Banner Sandbox*/}
-            <h2 id="component" style={{display: "none"}}>Component</h2>
-            <Sandbox properties={heroBannerBindings} onChange={onSandboxChange}>
+            <h2 id="component" style={{ display: "none" }}>
+              Component
+            </h2>
+            <Sandbox properties={heroBannerBindings} fullWidth={true} onChange={onSandboxChange}>
               <GoAHeroBanner {...heroBannerProps}>
-                Resources are available to help Alberta entrepreneurs and small businesses start, grow
-                and succeed.
+                <div>
```

Comment:
Can you remove this `div`? It may imply that it is required. It also differs from the examples below.